orderbook/src/api/filter.rs

```diff
@@ -18,16 +20,10 @@ fn extract_user_order() -> impl Filter<Extract = (OrderCreation,), Error = warp:
     warp::body::content_length_limit(MAX_JSON_BODY_PAYLOAD).and(warp::body::json())
 }
 
-fn extract_sell_token(
-) -> impl Filter<Extract = (handler::FeeRequestBody,), Error = warp::Rejection> + Clone {
-    // (rejecting huge payloads)...
-    warp::body::content_length_limit(MAX_JSON_BODY_PAYLOAD).and(warp::body::json())
-}
-
 pub fn create_order(
     orderbook: Arc<OrderBook>,
-) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
-    warp::path!("api" / "v1" / "orders")
+) -> impl Filter<Extract = (WithStatus<Json>,), Error = warp::Rejection> + Clone {
+    warp::path!("orders")
```

Comment:
All of the `Extract = impl warp::Reply` -> `Extract = (WithStatus<Json>,)` aren't necessary`.